<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-05T00:21:37 IST" name="Master Suite" finished-at="2025-09-05T00:22:23 IST" duration-ms="45545">
    <groups>
      <group name="Master">
        <method signature="TC005_Editaccountinfo.verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" name="verify_Checkout" class="testCases.TC005_Editaccountinfo"/>
      </group> <!-- Master -->
      <group name="Sanity">
        <method signature="TC005_Editaccountinfo.verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" name="verify_Checkout" class="testCases.TC005_Editaccountinfo"/>
      </group> <!-- Sanity -->
      <group name="DataDriven">
        <method signature="TC005_Editaccountinfo.verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" name="verify_Checkout" class="testCases.TC005_Editaccountinfo"/>
      </group> <!-- DataDriven -->
      <group name="Regression">
        <method signature="TC005_Editaccountinfo.verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" name="verify_Checkout" class="testCases.TC005_Editaccountinfo"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2025-09-05T00:21:37 IST" name="Test" finished-at="2025-09-05T00:22:23 IST" duration-ms="45545">
      <class name="testCases.TC005_Editaccountinfo">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" started-at="2025-09-05T00:21:38 IST" name="setup" finished-at="2025-09-05T00:21:43 IST" duration-ms="5631" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windoes]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" started-at="2025-09-05T00:21:45 IST" name="verify_Checkout" data-provider="LoginData" finished-at="2025-09-05T00:21:50 IST" duration-ms="4848" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[31@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Bharath]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Checkout -->
        <test-method signature="verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" started-at="2025-09-05T00:21:50 IST" name="verify_Checkout" data-provider="LoginData" finished-at="2025-09-05T00:22:02 IST" duration-ms="11816" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[23@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Bharath]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testCases.TC005_Editaccountinfo.verify_Checkout(TC005_Editaccountinfo.java:51)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Checkout -->
        <test-method signature="verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" started-at="2025-09-05T00:22:02 IST" name="verify_Checkout" data-provider="LoginData" finished-at="2025-09-05T00:22:13 IST" duration-ms="11364" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[22@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Bharath]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testCases.TC005_Editaccountinfo.verify_Checkout(TC005_Editaccountinfo.java:51)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Checkout -->
        <test-method signature="verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" started-at="2025-09-05T00:22:14 IST" name="verify_Checkout" data-provider="LoginData" finished-at="2025-09-05T00:22:18 IST" duration-ms="4241" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[31@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Bharath]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Checkout -->
        <test-method signature="verify_Checkout(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" started-at="2025-09-05T00:22:18 IST" name="verify_Checkout" data-provider="LoginData" finished-at="2025-09-05T00:22:22 IST" duration-ms="4393" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[31@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Bharath]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_Checkout -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC005_Editaccountinfo@134d26af]" started-at="2025-09-05T00:22:22 IST" name="tearDown" finished-at="2025-09-05T00:22:23 IST" duration-ms="353" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC005_Editaccountinfo -->
    </test> <!-- Test -->
  </suite> <!-- Master Suite -->
</testng-results>
